### DISCUSSION TITLE
对于软件的建议

### DISCUSSION THREAD

**Discussion by ATMfeng** [ywmoyue/biliuwp-lite#351] [General]
### 问题描述

大佬能不能维护一下3.10的版本

### 描述解决方案

3.10版本的哔哩哔哩动画UWP已经年久失修了，投搞，动态都打不开了。

### 代码贡献

是，我愿意提交代码

**Comment by GD-Slime** [comment]
你说的是这个吗? https://apps.microsoft.com/detail/%E5%93%94%E5%93%A9%E5%93%94%E5%93%A9uwp%E7%AC%AC%E4%B8%89%E6%96%B9/9N7C87236453?hl=en-us&gl=US

如果是的话，这个版本最初是 @xiaoyaocz 逍遥橙子大佬的v3版本。这个版本已经在很久之前弃用，转而更新v4版本了。
现在这个项目是对v4版本的继续维护。
一般来说，维护v3版本的意义不大而所需的成本也很多，应该不会去动了。 

你可以在release中下载本项目维护的最新版本，如果有在v3中有而v4中没有的功能欢迎提出或贡献代码。

**Comment by ywmoyue** [comment]
非本项目问题，移动至discussion

**Comment by ywmoyue** [comment]
我以前编译过一个版本解决了登录问题，不知道还能不能用 https://github.com/ywmoyue/biliuwp/releases/tag/v3.12.0
